Wrong checksums for opam

Hi,

The checksums (hashes, really) for opam don't match those in the archive:

```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - opam-client.2.0.1: Bad checksum
          - opam-core.2.0.1: Bad checksum
          - opam-format.2.0.1: Bad checksum
          - opam-repository.2.0.1: Bad checksum
          - opam-solver.2.0.1: Bad checksum
          - opam-state.2.0.1: Bad checksum
```

Thanks!

--- a/packages/opam-client/opam-client.2.0.1/opam
+++ b/packages/opam-client/opam-client.2.0.1/opam
@@ -30,8 +30,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-core/opam-core.2.0.1/opam
+++ b/packages/opam-core/opam-core.2.0.1/opam
@@ -32,8 +32,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-devel/opam-devel.2.0.1/opam
+++ b/packages/opam-devel/opam-devel.2.0.1/opam
@@ -37,8 +37,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-format/opam-format.2.0.1/opam
+++ b/packages/opam-format/opam-format.2.0.1/opam
@@ -28,8 +28,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-installer/opam-installer.2.0.1/opam
+++ b/packages/opam-installer/opam-installer.2.0.1/opam
@@ -29,8 +29,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-repository/opam-repository.2.0.1/opam
+++ b/packages/opam-repository/opam-repository.2.0.1/opam
@@ -27,8 +27,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-solver/opam-solver.2.0.1/opam
+++ b/packages/opam-solver/opam-solver.2.0.1/opam
@@ -30,8 +30,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

--- a/packages/opam-state/opam-state.2.0.1/opam
+++ b/packages/opam-state/opam-state.2.0.1/opam
@@ -27,8 +27,8 @@ dev-repo: "git+https://github.com/ocaml/opam.git"
 url {
   src: "https://github.com/ocaml/opam/archive/2.0.1.tar.gz"
   checksum: [
-    "md5=91a3b7aff1970692ba77aafb92d192d1"
-    "sha512=ec18d7f5c8b84303bb12a85baff7140b315b551a0c25e07c712c7c6368c8f3e359706590146c8f9248a7c34e35689ade06b3ef57f0562cecb5a86685438b4d8c"
+    "md5=1e55c234fa09e678e8115a31f3ec2a05"
+    "sha512=d71c362d70a6b6cb9fe54cc43bfbea8079cca05d4721a7dda951196f543a1ff5af81399a246e0cd68bc590c7bdca388d96c7127217ed637261957047ce725516"
   ]
 }
 description:"""

